lib/gameteam.rb

```diff
@@ -65,6 +65,66 @@ def self.fewest_tackles_by_season(season_id)
     GameTeam.fewest_tackles(tackles_per_team_hash)
   end
 
+  def self.winningest_coach(season_id)
```

Comment:
I like these methods for checking the most winningest coach as well as the worst coach!